.golangci.yml

```diff
@@ -71,18 +71,12 @@ linters:
         - deferInLoop
         - dupArg
         - exitAfterDefer
-        - exposedSyncMutex
         - hugeParam
         - importShadow
-        - nestingReduce
-        - paramTypeCombine
```

Comment:
I think this is a bad linter and I would not want to enable it. `(a string, b string, ...)` is A LOT easier to read than dropping repeated types. With explicit types it's obvious, with combined types I need to scan more of the string to figure out the types, and the use of commas isn't helping.